Layers/LayerComposition.gd

```diff
@@ -61,6 +61,7 @@ class UIInfo:
 # RenderInfo data classes
 class RenderInfo extends RefCounted:
 	var renderer = null
+	var render_scene: Node3D
```

Comment:
wording issue: actually the `renderer` variable above stores the _scene_, whereas this variable stores the _instanced node_, right? so this should be renamed to `renderer_instance` if I am not mistaken